app/src/main/java/at/roteskreuz/stopcorona/model/repositories/ExposureNotificationRepository.kt

```diff
@@ -87,6 +98,10 @@ interface ExposureNotificationRepository {
      * Get the current (refreshed) state of the Exposure Notifications Framework state.
      */
     suspend fun isAppRegisteredForExposureNotificationsCurrentState(): Boolean
+
+    fun getTemporaryExposureKeys()
```

Comment:
Missing Kdoc. Getter without returning value. If this is a call for a get event which will be observed in `observeTemporaryExposureKeys`, please use prefix `fetch` and describe where the result will be observed.